apps/toolkit-app/pages/auth/forgot-password.tsx

```diff
@@ -1,15 +1,15 @@
-import { BlitzPage, useMutation } from "blitz"
 import Layout from "app/core/layouts/Layout"
+import { useState } from "react"
 import { LabeledTextField } from "app/core/components/LabeledTextField"
 import { Form, FORM_ERROR } from "app/core/components/Form"
 import { ForgotPassword } from "app/auth/validations"
 import forgotPassword from "app/auth/mutations/forgotPassword"
+import { invoke } from "@blitzjs/rpc"
```

Comment:
Should be replaced with useQuery after https://github.com/blitz-js/blitz/pull/3302 is merged